docs/manager.md

```diff
@@ -24,20 +24,20 @@ The future owners of protocol on whose success the experiment of running a decen
 
 
 
-As the intialization function call can be seen above the manager is expected to provide certain paramters as:
+As the intialization function call can be seen above the manager is expected to provide certain paramters like:
```

Comment:
As the initialization function call can be seen above the manager is expected to provide certain parameters like: